search-app/src/cmr/search/services/parameters/converters/orbit_number.clj

```diff
@@ -27,13 +27,14 @@
       (if value
         (qm/map->OrbitNumberValueCondition numeric-map)
         (qm/map->OrbitNumberRangeCondition numeric-map)))
-    (catch NumberFormatException e
-      (errors/internal-error! msg/non-numeric-orbit-number-parameter))))
+    (catch NumberFormatException _e
+      ;; TODO: "pull-before-flight": send to bamboo for testing, why not a function ; kondo suggestion
```

Comment:
I'm removing this comment in the next version (I assume changes will be suggested). It is here to mark what I thought was a strange clj-kondo change that I wanted tested in bamboo.